vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -208,6 +215,14 @@ def process_weights_after_loading(self, layer: Module) -> None:
             else:
                 layer.input_scale = None
 
+            # If rocm, adjust the scaling factor
+            if isinstance(current_platform, rocm.RocmPlatform):
+                layer.weight_scale = Parameter(layer.weight_scale * 2,
+                                               requires_grad=False)
+                if layer.input_scale is not None:
+                    layer.input_scale = Parameter(layer.input_scale * 2,
+                                                  requires_grad=False)
```

Comment:
If you could leave a comment describing a reference for the `* 2`, would be appreciated